configure.ac

```diff
@@ -523,7 +524,9 @@ fi
 
 dnl swig stuff
 if test -z "$ENABLE_SOS_TRUE"; then
-	which cython >/dev/null 2>&1 || AC_MSG_ERROR("cython not found")
+	if test -z "$ENABLE_LIB_PYMAP_TRUE"; then
+		which cython >/dev/null 2>&1 || AC_MSG_ERROR("cython not found")
+	fi 
 	AX_PYTHON_DEVEL()
```

Comment:
So this should all be unnecessary.

Reply:
All of the Python3 support, which is being back ported from master to OVIS-4 requires Cython.

Reply:
Hi @baallan , all of the Python3 support, which is being back ported from master to OVIS-4 requires Cython.

Reply:
so ldms use of cython will be independent of sos? In which case the backport will be adding configure tests for cython? If yes to both, Sara can revise this to just delete the cython check. Then when the backport is merged things will continue to be fine.

Reply:
Yes, the Python API that was formerly generated using SWIG has been re-implemented in Cython for Python3.